### ISSUE TITLE
Commenting out lines breaks paragraphs

### ISSUE THREAD

**Issue by joshuahhh** [uwdata/living-papers#92]
(Pretty sure this is a Pandoc-Markdown issue, not LP, but it's annoying enough to me that I wouldn't mind fixing it in LP.)

I split my paragraphs into one-sentence-per-line:
```
Sentence 1.
Sentence 2.
Sentence 3.
```
I comment out the second sentence:
```
Sentence 1.
<!-- Sentence 2. -->
Sentence 3.
```
Now it's formatted as two paragraphs! No good.

**Comment by jheer** [comment]
Yes, this has burned me before, too, and arises from the Pandoc parser output. I will need to take a closer look to figure out if the Pandoc AST has the right info in it to know when and when not to "stitch" things back together into a single paragraph.
